src/System.Management.Automation/engine/hostifaces/MshHostUserInterface.cs

```diff
@@ -1120,13 +1099,14 @@ internal void FlushContentToDisk()
                 {
                     if (_contentWriter == null)
                     {
+                        var encoding = Utils.GetEncoding(this.Path);
                         try
                         {
```

Comment:
If we remove GetEncoding() we should put StreamReader to try block.

Reply:
Done as proposed